firmware/hw_layer/ports/stm32/stm32f4/mpu_util.cpp

```diff
@@ -77,6 +78,40 @@ EXTERNC int getRemainingStack(thread_t *otp) {
 
 #endif /* GNU / IAR */
 
+
+#if (EFI_FLASH_SPI == TRUE)
```

Comment:
this stuff can probably move to `stm32_common.cpp` - it should work the same on f7

Reply:
Currently ext flash is enabled (and tested) only on F4. Later we can move it to common files.